### ISSUE TITLE
I can't deploy projects.

### ISSUE THREAD

**Issue by ejherran** [kobotoolbox/kobo-docker#165]
Morning.

I have followed the installation steps to the letter for both server and local mode, but I always get an error when trying to deploy a project.

The error message says "No JSON object could be decoded".

That's all I get, even with "KPI_DJANGO_DEBUG=True" and "KOBOCAT_DJANGO_DEBUG=True", nothing appears in the logs or console.

Using the issue_222 image, I manage to deploy the project, but the forms cannot be used.

I have read and tried everything I could find about this mistake, but nothing has helped me.

Please, I need help. I need to get this tool in production as soon as possible.

Oh, thank you.

**Comment by sbpipb** [comment]
I am experiencing the same issue as well. Using the latest image.

**Comment by ejherran** [comment]
Yes, my only solution was to install using issue_222, then step by step jump to the latest version.

However issue_222 has been removed from the repository, making it impossible to replicate this method right now.
